### PR TITLE
Fix hyperlinks in Documentation Framework

### LINKED ISSUE #7479: [Docs] Fix links to be hyperlinked
#### Current State
Screenshot 2023-04-05 at 10 20 21 am

Screenshot 2023-04-05 at 10 06 06 am

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by goldfishdolphin**
**Notes for Reviewers**

This PR fixes #7479 

Screenshot 2023-04-05 at 4 29 53 pm
**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5be05bd5665cde90df0be0dada26b6bc31eb8292 at: https://642d95b29a77225a1dbdfc66--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a8cd66e2a03496eaeeeafba5dbd1f9888d612504 at: https://642d97a992354e005b693193--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5e3141fb38c4cb0feeee753845cc7de23b93ca33 at: https://642ddfbfefd06e00abe962c1--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Before contributing, please review the [Documentation Contribution Flow](https:/
...
 Meshery documentation is made of these components:
 
-- Framework - Jekyll
-- Theme - https://github.com/vsoch/docsy-jekyll
-- Repo - https://github.com/layer5io/meshery/tree/master/docs
-- DNS - https://meshery.layer5.io/docs
+- Framework - [Jekyll](https://jekyllrb.com)
+- Theme - [https://github.com/vsoch/docsy-jekyll](https://github.com/vsoch/docsy-jekyll)
+- Repo - [https://github.com/layer5io/meshery/tree/master/docs](https://github.com/layer5io/meshery/tree/master/docs)
+- DNS - [https://meshery.layer5.io/docs](https://meshery.layer5.io/docs)
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:22]:
```suggestion
- Site - [https://docs.meshery.io](https://docs.meshery.io)
```

**Review by leecalcote — COMMENTED** [review]
This is great to seen, @goldfishdolphin.

**Comment by l5io** [comment]
🚀 Preview for commit 4dadcaf469907654b2409a5798cbe441365f32e2 at: https://642edb9b6b210a0a3a95b2a1--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Looks great, @goldfishdolphin. 👍

**Pull request merged**